### DISCUSSION TITLE
Are you using QtRvSim at your organization? Tell us!

### DISCUSSION THREAD

**Discussion by jdupak** [cvut/qtrvsim#136] [Q&A]
It is nice to know that our work is useful to others and it can help us when presenting the project to get new developers and funding.

**Comment by petrsocha** [comment]
Telling you that we are using QtRvSim at Střední průmyslová škola Liberec, in the microarchitecture course for IT students :-). Good job on the simulator!

**Reply by ppisa** [reply]
By the way, if you need teaching materials for related computer architecture courses, then there are released our basic course slides CC-BY-SA licensed sources https://github.com/cvut/comparch-slides in Czech and English language. reuse, feedback or even corrections and contributions are welcomed... The formatted PDFs can be found there https://cw.fel.cvut.cz/wiki/courses/b35apo/lectures/start for Czech version and there https://cw.fel.cvut.cz/wiki/courses/b35apo/en/lectures/start for English one
